Cake.Recipe/Content/build.cake

```diff
@@ -132,10 +127,7 @@ BuildParameters.Tasks.DotNetCoreRestoreTask = Task("DotNetCore-Restore")
 {
     DotNetCoreRestore(BuildParameters.SolutionFilePath.FullPath, new DotNetCoreRestoreSettings
     {
-        Sources = new [] {
-            "https://api.nuget.org/v3/index.json",
-            "https://www.myget.org/F/cake-contrib/api/v3/index.json"
-        },
+        Sources = BuildParameters.NuGetSources,
```

Comment:
Well I removed the hard coded ones and made them configurable, AFAIK it does support V2 feeds, no V2 feeds specified anywhere in Cake Recipe?